src/braket/flake8_plugins/braket_checkstyle_plugin.py

```diff
@@ -87,6 +87,7 @@ class _Visitor(ast.NodeVisitor):
         "BCS020": "Function '%s' has return documentation but no return type.",
         "BCS021": "Function '%s' is missing return documentation.",
         "BCS022": "Found '%d' invalid indents starting with line ('%s').",
+        "BCS023": "Argument '%s' is optional but type hint doesn't end with '| None'.",
```

Comment:
I'd prefer we avoid "optional" here as it is ambiguous and doesn't exactly align with the requirements. `bool = False` or `str = ''` are both optional args that don't include `None` in their type. It's an existing pain point that `Optional[...]` doesn't align with whether an arg is "optional", so might as well use clearer terms

Reply:
Good call. Will change.